authors/apps/comments/views.py

```diff
@@ -125,3 +128,49 @@ def get(self, request, id, *args, **kwargs):
         serializer = self.serializer_class(thread, many=True)
         return Response({"comments": serializer.data,
                          "thread count": thread.count()})
+
+
+class LikeCommentApiView(generics.RetrieveUpdateAPIView):
+
+    permission_classes = (IsAuthenticated,)
+    serializer_class = serializers.LikeCommentSerializer
+    renderer_classes = (LikeComementsJSONRenderer, )
+    queryset = LikeComment.objects.all()
+
+    def get_queryset(self):
+        return LikeComment.objects.all()
+
+    def retrieve(self, request, *args, **kwargs):
+        likes = self.get_queryset()
+        serializer = self.serializer_class(data=likes, many=True)
+        serializer.is_valid()
+        return Response(serializer.data)
+
+    def update(self, request, *args, **kwargs):
+        id = self.kwargs.get('id')
+        comment = get_object_or_404(
```

Comment:
@Opio-Emmanuel-Omona You should use the get_object method as suggested during the review session so as each method does what its supposed to do.

Reply:
@archbishop12 I have included the get_object() method